_posts/2023-07-30-Server-Metadata.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Adding metadata to Servers"
```

Comment:
```suggestion
title: "Allow or block bots"
```

I hope the title convention in this file supports _sentence casing_ because that's my jam. I also know you often use _-ing_ convention, but this does not seem to be the title convention in this file, thank goodness.

If you want to introduce the generalized architecture, I won't stand in your way. I suggest just covering the one bot convention in this file, in a much shorter article. I would volunteer, except I believe there are adequate bot controls now. Don't introduce admincontact. We should brainstorm additional metadata that might also make sense to introduce, but I'm not aware of anything yet.